lib/group/group.routes.js

```diff
@@ -0,0 +1,69 @@
+const Joi = require('joi');
+
+
+module.exports = {
+  routes: (models) => {
+    // const controllers = require('./group.controllers')(models);
+    const controllers = require('./group.controllers')(models);
+    const groupModels = require('./group.models');
+
+    return [
+      {
+        method: 'GET',
+        path: '/groups',
+        // handler: controllers.viewAllGroups,
+        // TODO: move this into controllers file and expand into real logic
+        handler: controllers.getGroups,
+        config: {
+          auth: 'jwt',
+          description: 'Get Groups',
+          notes: 'Returns all groups.',
+          tags: ['api', 'Groups'],
+          validate: {
+            query: groupModels.apiFilterQuery,
+          }
```

Comment:
removing apiFilterQuery functionality for MVP should also entail commenting out this validation so the api doesn't claim to have this functionality. It will also probably fail, since this filter model is no longer defined.